### ISSUE TITLE
Codekit slowly loads HTML, but not loading CSS

### ISSUE THREAD

**Issue by kylebahr** [bdkjones/CodeKit#506]
**Quick, short summary:**

First things first: I've used Codekit w/ success in the past. This issue just started happening. I'm using Desktop Server for local Wordpress development, so I have an external server set in the Browser-refreshing settings. Recently, my stylesheet stopped loading when I try to preview using Codekit. HTML is loading, but very slowly. I'm using SASS, and my SASS file is compiling correctly. Been through all the troubleshooting issues on CK's site, but still nothing.  

**Expected results:**

Setup site using Desktop Server, set external server settings in Codekit, website previews and refreshes automatically.

**Actual results:**

Setup site using Desktop Server, set external server settings in Codekit, website HTML loads after about 30 seconds, CSS never loads. Have tried in Chrome, Firefox and Safari. 

**Your configuration (any details about your system that you think might be relevant)**

2015 Macbook Pro running Sierra 10.12, using Desktop Server for local Wordpress development.

Thanks for any help on this! I'm stumped

**Comment by th3car3tak3r** [comment]
Bryan, thanks for your response. I tested with a non-Wordpress site and the communication works fine. Do you have any ideas how to stop Wordpress from redirecting traffic in an infinite loop?

**Comment by bdkjones** [comment]
I don’t recall the settings off hand, but it’s a common problem with WordPress sites in development environments. A little googling should turn up what you need to adjust in WordPress.

Also: try Mamp as your external server. Nothing ever seems to be as reliable as Mamp.

**Comment by benediktdertinger** [comment]
@th3car3tak3r did you find any solution to the problem? I was stuck on this for quite some time with WP projects.

I got very slow page reloads and it seemed that on some SCSS changes (appeared randomly) the whole page got refreshed (for a short moment only HTML was shown – flickering – and then my styles reappeared) instead of css injected.

I then changed some things and can't currently reproduce what has been the exact trigger of fixing it but it might help others:
1. I switched to a `.test `TLD instead of using public ones before ([mind that .dev needs tls](https://github.com/bdkjones/CodeKit/issues/420))). Don't forget to adapt the fields `siteurl` and `home` in **wp_options** table.
2. I recreated all TLS certificates (MAMP Pro and CK) **and removed the old ones (keychain)!**
3. CK was alrady setup to watch only the theme folder but I removed several additional folders as well
4. Finally this very long lasting problem got resolved and style changes were injected via formerly known css animations and no reload.

Hope this helps!
